pkg/reconciler/tenancy/clusterworkspaceshard/clusterworkspaceshard_controller.go

```diff
@@ -121,7 +121,7 @@ func (c *Controller) processNextWorkItem(ctx context.Context) bool {
 
 	logger := logging.WithQueueKey(klog.FromContext(ctx), key)
 	ctx = klog.NewContext(ctx, logger)
-	logger.Info("processing key")
+	logger.V(1).Info("processing key")
```

Comment:
V(1) or V(4), seeing both in different controllers.

Do we have written down our guideline for the levels? We discussed in Slack.

Reply:
I wrote your comments into the epic - you said 1

Reply:
1 is fine, just noticed that we have multiple variants :)

Reply:
Will need a second pass to catch it ... or some linter rule :)